### DISCUSSION TITLE
Job offer:
Looking for a programmer to create a DMA cheat for various games

### DISCUSSION THREAD

**Discussion by MemoryWhispererDMA** [Rakeshmonkee/DMA#7] [General]
-Experience in Creating DMA cheats?
-You are above 18 y/o
-You want another Stable income
-Willing to create Updates for the Software and work with us Longtherm?

We offer payment via Bank transfer PayPal and Crypto

Just react to this post and we gonna connect

Have a great Day everyone, wish you all the best

**Comment by ggbond190** [comment]
How to cooperate

**Reply by liu20080602** [reply]
u can？
